drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -1446,3 +1457,300 @@ def run_quality_analysis(total_obj_list, run_compare_num_sources=True, run_find_
+                                    'distribution_characterization_statistics.Y_offset',
+                                    'distribution_characterization_statistics.Y_standard_deviation',
+                                    'distribution_characterization_statistics.maximum_neighbor_distance',
+                                    'distribution_characterization_statistics.mean_neighbor_distance',
+                                    'distribution_characterization_statistics.minimum_neighbor_distance',
+                                    'distribution_characterization_statistics.standard_deviation_of_neighbor_distances']
+
+    gaia_cols = get_pandas_data(data_source, gaia_col_names)
+
+    gaia_plots = build_gaia_plots(gaia_cols, gaia_col_names, 
+                                  output_basename=output_basename)
+    
+    # Generate plots for point-segment catalog cross-match comparisons
+    xmatch_col_names = HOVER_COLUMNS + ['Cross-match_details.number_of_cross-matches',
+                                        'Cross-match_details.point_catalog_filename',
+                                        'Cross-match_details.point_catalog_length',
+                                        'Cross-match_details.point_frame',
+                                        'Cross-match_details.segment_catalog_filename',
+                                        'Cross-match_details.segment_catalog_length',
+                                        'Cross-match_details.segment_frame',
+                                        'Cross-matched_point_catalog.Declination',
+                                        'Cross-matched_point_catalog.Right ascension',
+                                        'Cross-matched_segment_catalog.Declination',
+                                        'Cross-matched_segment_catalog.Right ascension',
+                                        'Segment_-_point_on-sky_separation_statistics.3x3_sigma-clipped_mean',
+                                        'Segment_-_point_on-sky_separation_statistics.3x3_sigma-clipped_median',
+                                        'Segment_-_point_on-sky_separation_statistics.3x3_sigma-clipped_standard_deviation',
+                                        'Segment_-_point_on-sky_separation_statistics.Non-clipped_max',
+                                        'Segment_-_point_on-sky_separation_statistics.Non-clipped_mean',
+                                        'Segment_-_point_on-sky_separation_statistics.Non-clipped_median',
+                                        'Segment_-_point_on-sky_separation_statistics.Non-clipped_min',
+                                        'Segment_-_point_on-sky_separation_statistics.Non-clipped_standard_deviation']
+
+    xmatch_cols = get_pandas_data(data_source, xmatch_col_names)
+
+    xmatch_plots = build_crossmatch_plots(xmatch_cols, xmatch_col_names, 
+                                  output_basename=output_basename)
+    
```

Comment:
The functions "build_*_plots()" do not actually return anything.

Reply:
They now return the name of the output plot generated by the function, since that is something that can be generated internally.  The docstrings were updated to reflect this change, along with making sure all the calling routines for all the build*plot functions were updated to accept the return value.  They currently don't do anything with those values, but now they could.